os/registry-authentication.md

```diff
@@ -0,0 +1,199 @@
+# Using authentication for a registry
+
+To pull images behind an authentication wall you need to setup some form of authentication with your image registry. These are either [quay.io][quay-site] or [hub.docker.com][docker-hub-site] or a self-hosted `registry.example.io/v0`.
+
+This document outlines the preferred method of authentication with an image registry like [quay.io][quay-site] and [hub.docker.com][docker-hub-site] and how setup registry auth for Docker, Kubernetes, rkt, and Mezos.
+
+## Using a Quay robot for registry auth
+
+The preferred way to setup registry authentication with [quay.io][quay-site] is to use a [Quay Robot][quay-robot]. This essentially acts as a login-token wth some nice features including:
+
+- Pre-populated repository-auth configuration files.
+- Credentials are limited to specific repositories.
+- Choose from read, write, or admin privileges.
+- Token regeneration.
+
+![Quay Robot settings][quay-bot-img]
+
+[comment]: # (Yes, this robot has been deleted. Un-obscured to help maintain educational value.)
+
+For more information, see the [Quay robot documentation][quay-robot].
+
+## Manual registry auth setup
+
+If you are using a registry other than Quay (e.g., Docker Hub, Docker Store, etc) you will need to manually configure your credentials with your container-runtime or orchestration tool.
+
+### Docker
+
+*Quay robots provide Docker registry authentication config files and intructions on to set them up. This can be found by accessing a robot's settings in the* **Robot Accounts** *tab under your user settings*.
+
+#### $HOME/.docker/config.json
```

Comment:
Can we make this describe the section better? "Writing the config file"? Do we even need this heading?